nautobot/dcim/tests/test_views.py

```
@@ -194,6 +194,14 @@ def setUpTestData(cls):
             "nestable": True,
         }
 
+        cls.bulk_edit_data = {
+            "description": "A generic description",
+            "comments": "Test LocationType",
```

Remove this nonexistent field? I'm going to open a housekeeping issue in the backlog to make tests fail if `form_data` or `bulk_edit_data` include fields that aren't present in the form/model, as currently this is just being silently ignored.

---

I updated for the content types but forgot the comments, my bad.